cmd/neofs-adm/internal/modules/morph/root.go

```diff
@@ -146,6 +150,36 @@ var (
 		},
 	}
 
+	mintBalanceCmd = &cobra.Command{
+		Use:   "mint-balance",
+		Short: "Mint new NEOFS tokens in the morph network",
+		PreRun: func(cmd *cobra.Command, _ []string) {
+			_ = viper.BindPFlag(alphabetWalletsFlag, cmd.Flags().Lookup(alphabetWalletsFlag))
```

Comment:
https://github.com/nspcc-dev/neofs-node/pull/2799#discussion_r1548117070 and https://github.com/nspcc-dev/neofs-node/issues/2800?

Reply:
I'm just doing what every other command does here. And the problem is that we can't return error.

Reply:
this func should not break so we can just exit/panic here